### PR TITLE
sig-tools: update meeting info and update member list

### PR DESCRIPTION
**Pull request by kennytm**
1. Included information of the regular meetings.
2. Updated member list to according to latest statistics

### CODE REVIEW

File: special-interest-groups/sig-tools/member-list.md [hunk 1]
```diff
@@ -33,6 +33,11 @@ Co-Leads:
 
 ## Active Contributors
 
+* [lance6716](https://github.com/lance6716)
```
Comment (IANTHEREAL) [inline special-interest-groups/sig-tools/member-list.md:36]: should we add `suzaku`?

File: special-interest-groups/sig-tools/member-list.md [hunk 2]
```diff
@@ -33,6 +33,11 @@ Co-Leads:
...
+
 ## Contributors
 
 * [AndrewDi](https://github.com/AndrewDi)
+* [glorv](https://github.com/glorv)
```
Comment (IANTHEREAL) [inline special-interest-groups/sig-tools/member-list.md:41]: what about `tsthght`, `djshow832`,`dixingxing0`, `freemindLi` from TiDB Binlog?

Comment (IANTHEREAL) [reply special-interest-groups/sig-tools/member-list.md:41]: `oscar-zyh` from ticdc

Comment (IANTHEREAL) [reply special-interest-groups/sig-tools/member-list.md:41]: `u5surf` from br

Comment (kennytm) [reply special-interest-groups/sig-tools/member-list.md:41]: these are the "1 PR contributors" i've mentioned in the description 😔

**Comment by kennytm** [comment]
@IANTHEREAL

Updated the list according to the stats gathered by @AndreMouche (thanks).

Note that @YuJunchen has been moved from "Committer" to "Reviewer" according to the stats, because he only has 22 PRs (needs 8 more), but this shouldn't have any effect on code contributions in the short term, since Reviewers still have the rights to LGTM.

File: special-interest-groups/sig-tools/README.md [hunk 3]
```diff
@@ -15,13 +15,14 @@ See [SIG Tools Member List](./member-list.md)
...
-There is no regular meeting.
+* Regular text-based meetings on Slack [#sig-tools] every two Thursdays at [13:00 UTC](http://www.thetimezoneconverter.com/?t=13:00&tz=UTC) since 2020 June 11th.
+* [Agenda, meeting notes and regulations](https://docs.google.com/document/d/1iKSi7JpWjODPTKOgc46N3BzuaH8R9XcDhLS0NhpcT9g/edit?usp=sharing).
 
 ## Contact
 
 Feel free to join our discussion if you are interested.
 
-* Slack: [#sig-tools](https://slack.tidb.io/invite?team=tidb-community&channel=sig-tools&ref=github_sig)
+* Slack: [#sig-tools]
```
Comment (AndreMouche) [inline special-interest-groups/sig-tools/README.md:25]: why?

Comment (kennytm) [reply special-interest-groups/sig-tools/README.md:25]: ?

The anchor is moved to line 39.

Comment (AndreMouche) [reply special-interest-groups/sig-tools/README.md:25]: OK

**Comment by IANTHEREAL** [comment]
LGTM

**Pull request merged**